client/src/App.js

```diff
@@ -1,19 +1,63 @@
-import React from "react";
-import { MuiThemeProvider } from "@material-ui/core";
-import { BrowserRouter, Route } from "react-router-dom";
+import React, { useCallback, useContext, useEffect } from 'react';
+import { MuiThemeProvider } from '@material-ui/core';
+import { BrowserRouter, Route } from 'react-router-dom';
+import { theme } from './themes/theme';
+import './App.css';
+import DashBoard from './pages/DashBoard';
+import api from './utils/api';
+import Navbar from './Components/Navbar';
+import Blog from './pages/Blog';
+import Faq from './pages/Faq';
+import Profile from './pages/Profile';
+import { StateProvider, store } from './context/store';
+import { AUTH_ERROR, USER_LOADED } from './context/types';
 
-import { theme } from "./themes/theme";
-import LandingPage from "./pages/Landing";
+function App() {
+  const { dispatch } = useContext(store);
+  const loadUser = useCallback(async () => {
+    try {
+      const res = await api.get('/users/auth');
```

Comment:
This doesn't work yet right?

Reply:
Correct, it's one of the reasons I have loadUser in the useEffect in an if statement

Reply:
Also, I haven't created the route on the backend, waiting for Nafis' pull